frameworks/kafka/universe/package.json

```diff
@@ -1,25 +1,27 @@
 {
-  "packagingVersion": "4.0",
-  "minDcosReleaseVersion": "1.9",
-  "upgradesFrom": [
-    "{{upgrades-from}}"
-  ],
+  "description": "Apache Kafka is used for building real-time data pipelines and streaming apps. It is horizontally scalable, fault-tolerant, wicked fast, and runs in production in thousands of companies.\n\nWhen deployed, this is going to start ",
   "downgradesTo": [
     "{{downgrades-to}}"
   ],
-  "name": "kafka",
-  "version": "{{package-version}}",
+  "framework": true,
   "maintainer": "support@mesosphere.io",
-  "description": "Apache Kafka",
+  "minDcosReleaseVersion": "1.10",
+  "name": "kafka",
+  "packagingVersion": "4.0",
+  "postInstallNotes": "The DC/OS Apache Kafka service is being installed.\n\n\tDocumentation: {{documentation-path}}\n\tIssues: {{issues-path}}",
+  "postUninstallNotes": "The DC/OS Apache Kafka service is being uninstalled.\n\nFor DC/OS versions from 1.10 no further action is required. For older DC/OS versions follow the instructions at {{documentation-path}}uninstall to remove any persistent state if required.",
+  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU, 2048 MB MEM and 1,5000 MB Disk.\n\nNote that Kafka requires a Zookeeper instance to be available before starting. It is a good practice to deploy a dedicated Zookeeper service, but you can always fall-back into using the DC/OS core Zookeeper instance instead.",
+  "scm": "https://github.com/mesosphere/dcos-kafka-service",
   "selected": true,
-  "framework": true,
   "tags": [
     "message",
     "broker",
     "pubsub",
     "kafka"
   ],
-  "preInstallNotes": "Default configuration requires 3 agent nodes each with: 1.0 CPU | 2048 MB MEM | 1 5000 MB Disk",
-  "postInstallNotes": "The DC/OS Apache Kafka service is being installed.\n\n\tDocumentation: {{documentation-path}}\n\tIssues: {{issues-path}}",
-  "postUninstallNotes": "The DC/OS Apache Kafka service is being uninstalled.\n\nFor DC/OS versions from 1.10 no further action is required. For older DC/OS versions follow the instructions at {{documentation-path}}uninstall to remove any persistent state if required."
+  "upgradesFrom": [
+    "{{upgrades-from}}"
+  ],
+  "version": "{{package-version}}",
+  "website": "https://docs.mesosphere.com/services/kafka/2.4.0-1.1.1"
```

Comment:
Should "website" be "https://docs.mesosphere.com/services/kafka"?

Reply:
When we release the docs for `2.4.0` (https://github.com/mesosphere/dcos-docs-site/pull/2029) this will be a valid URL. I was thinking to pin it to the version we are releasing.